dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/udp/UDPConfigurationImpl.java

```diff
@@ -83,13 +85,17 @@ public void applyConfiguration(Bootstrap bootstrap) {
         bootstrap.option(ChannelOption.SO_REUSEADDR, true);
 
         if ((getReceiveBufferSize() >= UDPConfigConstants.RECEIVE_BUFFER_SIZE_MIN)
```

Comment:
If getReceiveBufferSize() is cheap then calling it 4 times in a blink of an eye might not add much time but, when writing middleware, if it was in the send receive path and not in bootstrap, and even though the JIT etc might be able to inline it, it is not a great look.

Reply:
5 times (missed one).